lib/player.rb

```diff
@@ -0,0 +1,61 @@
+require 'awesome_print'
+require 'pry'
+require_relative 'scoring'
+
+module Scrabble
+  class Player < Scoring
+
+    attr_reader :name, :played_words, :player_tiles
+    attr_writer :total_score
+
+    def initialize(name)
+      @player_tiles = []
+      @name = name
+      @played_words = []
+      @total_score = 0
+    end
+
+    def draw_tiles(tile_bag)
+      tile_bag = Scrabble::TileBag.new
+      @player_tiles = tile_bag.draw_tiles(7 - @player_tiles.length)
+      tiles
+    end
+
+    def tiles
+      return @player_tiles
+    end
+
+    def play(word)
+      if @total_score >= 100
+        return false
+      else
+        @played_words << word
```

Comment:
You should make sure the word can be scored before adding it to the list of played words.  What if the player tries to play `!@#!@`?